web/src/main/java/org/cbioportal/web/error/GlobalExceptionHandler.java

```diff
@@ -177,4 +177,12 @@
         ErrorResponse response = new ErrorResponse("Specified token cannot be found");
         return new ResponseEntity<>(response, HttpStatus.NOT_FOUND);
     }
+
+    @ExceptionHandler(CacheNotFoundException.class)
+    public ResponseEntity<ErrorResponse> handleCacheNotFoundException(CacheNotFoundException ex) {
+
```

Comment:
extra line here :)

Reply:
Ok .. I will clean it up here, and also in the other handler declarations above.

Reply:
done